docs/admin-ports.md

```diff
@@ -38,6 +38,8 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
+ - 15101: dcos-marathon libprocess
+ - 15201: dcos-metronome libprocess
```

Comment:
@spahl and I chatted and we are ok leaving these ports as is, there is not much value at this point in trying to use a contiguous range since master ports are already all over the place. I did verify that 15101 and 15201 are unassigned in the IANA services and port registry.